src/scripts/main.js

```diff
@@ -1,3 +1,63 @@
 'use strict';
 
-// write code here
+const table = document.getElementsByTagName('table');
+
+const rowsBody = table[0].tBodies[0].rows;
+const rowsHead = table[0].tHead.rows;
+
+for (const item of rowsHead[0].cells) {
+  const spanElem = document.createElement('span');
+
+  spanElem.textContent = item.innerText;
+  item.firstChild.nodeValue = '';
+  item.prepend(spanElem);
+}
+
+document.addEventListener('click', (e) => {
+  let sort = [];
+
+  if (e.target.tagName !== 'SPAN') {
+    return;
+  }
+
+  switch (e.target.textContent) {
+    case 'Name':
+      sort = Sort([...rowsBody], 0);
```

Comment:
```suggestion
      sort = Sort([...rowsBody], 'name');
```
better to use somth more meaningfull